turbopack/crates/turbopack-ecmascript/tests/tree-shaker/analyzer/1/output.md

```diff
@@ -593,10 +593,10 @@ foobarCopy += "Unused";
 import { d as foobar } from "__TURBOPACK_PART__" assert {
     __turbopack_part__: -7
 };
-import { e as upper } from "__TURBOPACK_PART__" assert {
-    __turbopack_part__: -6,
-    __turbopack_original__: "module"
+import "__TURBOPACK_PART__" assert {
+    __turbopack_part__: 5
 };
+import { upper } from "module";
```

Comment:
Fine to merge this way, but as follow-up we need to get rid of the part 5 module `import "module";` completely